pkg/operation/botanist/component/kubeapiserver/hvpa.go

```diff
@@ -125,7 +125,7 @@ func (k *kubeAPIServer) reconcileHVPA(ctx context.Context, hvpa *hvpav1alpha1.Hv
 			},
 			ScaleDown: hvpav1alpha1.ScaleType{
 				UpdatePolicy: hvpav1alpha1.UpdatePolicy{
-					UpdateMode: &scaleDownUpdateMode,
+					UpdateMode: &updateModeAuto,
```

Comment:
Please add a short comment (few words) that HPA does not work with mode `Off` and needs to be always `Auto` even though scale down is disabled.

Reply:
Done in https://github.com/gardener/gardener/commit/aff93c9d6f639aa582efa44d61826dadffb86b8e

Reply:
```suggestion
					UpdateMode: &updateModeAuto, # HPA does not work with update mode 'Off' and needs to be always 'Auto' even though scale down is disabled.
```